packages/skill-lesson/hooks/use-mux-player.tsx

```diff
@@ -82,15 +83,18 @@ export const VideoProvider: React.FC<
       })
     : null
 
-  const {data: abilityRules, status: abilityRulesStatus} =
-    trpcSkillLessons.modules.rules.useQuery({
-      moduleSlug: module.slug.current,
-      moduleType: module.moduleType,
-      lessonSlug: exerciseSlug,
-      sectionSlug: section?.slug,
-      isSolution: lesson._type === 'solution',
-      convertkitSubscriberId: subscriber?.id,
-    })
+  const {
+    data: abilityRules,
+    status: abilityRulesStatus,
+    refetch: refetchAbility,
+  } = trpcSkillLessons.modules.rules.useQuery({
+    moduleSlug: module.slug.current,
```

Comment:
this is what `canShowVideo` var is based off of and since it's responsible for the blocked overlay it needs to be refetched once claiming a seat has been successful. I tried invalidating `getAbilities` route but that didn't do the trick.

Reply:
i've noticed that invalidate doesn't automatically trigger a rerender but makes sure that it's fetched next render